blocklylib-core/src/main/java/com/google/blockly/android/ui/VirtualWorkspaceView.java

```diff
@@ -159,6 +161,11 @@ public boolean isScrollable() {
         return mScrollable;
     }
 
+    public void setZoomBehavior(int zoomBehavior){
+        setScrollable(zoomBehavior > WorkspaceFragment.ZOOM_BEHAVIOR_FIXED);
+        setScalable(zoomBehavior > WorkspaceFragment.ZOOM_BEHAVIOR_BUTTONS_ONLY);
+    }
```

Comment:
Wrap these comparisons in new static methods ZoomBehavior.isScrollable(value) and ZoomBehavior.isPinchZoomEnabled(value).